### ISSUE TITLE
GitHub warning about bans - what they actually meant

### ISSUE THREAD

**Issue by Alexander01998** [Alexander01998/totally-not-youtube-dl#2]
GitHub's actual warning can be found here: https://github.com/github/dmca#contributing

In the last paragraph, they said the following:

> If you commit or post content **to this repository** that violates our Terms of Service, we will delete that content and may suspend access to your account as well.

(By "this repository" they meant the [github/dmca](https://github.com/github/dmca) repo.)

This was posted in response to
1. people flooding the DMCA repo with fake Pull Requests like https://github.com/github/dmca/pull/8170 and https://github.com/github/dmca/pull/8153
2. people using [this exploit](https://gist.github.com/lrvick/02088ee5466ca51116bdaf1e709ddd7c) to upload copies of youtube-dl directly to the dmca repo without permission ([example](https://github.com/github/dmca/tree/416da574ec0df3388f652e44f7fe71b1e3a4701f)).

GitHub never said that they would ban anyone for hosting copies of youtube-dl on their own repositories. GitHub actually has a [policy of keeping forks](https://docs.github.com/en/free-pro-team@latest/github/site-policy/dmca-takedown-policy#b-what-about-forks-or-whats-a-fork) of DMCA'd repos up unless they are specifically listed in the takedown notice.

Several news outlets have over-exaggerated GitHub's warning:

- https://torrentfreak.com/github-warns-users-reposting-youtube-dl-they-could-be-banned-201102/  
   Clickbait title, but once you read the whole article it becomes clear what they actually meant. It's a good article and I recommend it, but you have to read the whole thing to understand it properly.

- https://www.xda-developers.com/github-warns-of-bans-for-users-still-distributing-youtube-dl/  
  These guys clearly didn't do their research. They say things like "the court ordered all instances of it to be taken down", which is completely made up. No court in the world has actually made a decision about this situation... yet. XDA also cite the TorrentFreak article as their source, but directly contradict it. Looks like they just read the title...

**TL;DR:** GitHub never said anything about users hosting forks of youtube-dl on their own accounts and organizations. There is absolutely no evidence that they would ban people for doing that.
